### ISSUE TITLE
Portuguese (pt-BR) translation

### ISSUE THREAD

**Issue by tmergulhao** [ncase/crowds#2]

**Comment by italohdc** [comment]
Can I help with the translation?

**Comment by tmergulhao** [comment]
Of course. I was planning on starting tomorrow because of the holiday. But feel free to take over, @italohdc :)

**Comment by riuri** [comment]
Check my fork too

**Comment by italohdc** [comment]
Nice @riuri :smile: 

I see you have translated some of the _Introduction_ section. Can you make a Pull Request to my fork, so I can merge what you have already done?

**Comment by riuri** [comment]
Yes, I can. Though I don't know if it's gonna be able to merge soon, but let's try

**Comment by riuri** [comment]
@italohdc Actually, I'd suggest it's feasible to check out each other translation status in order to avoid repeated work

**Comment by riuri** [comment]
After translating, we have to check whether all the text got into the page. I noticed some ends of the texts are missing once you open `pt.html` in the browser.

**Comment by italohdc** [comment]
@riuri Just checked what you have done so far and merged your commits. I think each one should select some different sections to translate and work in the same fork.

---

We can translate the html file and after we finish, we make a single review of the page in the browser to see if everything is okay and fix any bug.

**Comment by riuri** [comment]
@italohdc I've added you as a collaborator in my fork, so you can push directly there.

-----------------------

I'd say I get the odd chapters 5 and 7 and you get the even chapters 4 and 6. Whoever finishes first gets the sandbox mode.

Definitely let's review the end result and make a pull request on ncase's repo once we've both reviewed it.

**Comment by riuri** [comment]
@italohdc nem vi onde acabava a 5, acabei traduzindo a 6 também.

**Comment by Ghust1995** [comment]
Are you done translating, can I help with something?

**Comment by italohdc** [comment]
@Ghust1995 We actually already finished translating. We are only gonna review it to make a Pull Request.

If you want to help us reviewing, it would be very helpful :D

**Comment by italohdc** [comment]
@riuri I think it's best to keep _"original in English"_ in English, as someone that doesn't speak portuguese will understand what's written

**Comment by riuri** [comment]
OK, let's keep "original in English", as other translations kept on that

**Comment by riuri** [comment]
@Ghust1995 I gave you push access to riuri/crowds. If you can add suggestions and reviews, please add some issues there, but keep in mind we are almost finished
